lib/include/tick/prox/prox_l1.h

```diff
@@ -32,9 +26,13 @@ class DLL_PUBLIC TProxL1 : public TProxSeparable<T> {
   T value_single(T x) const override;
 
  public:
-  TProxL1(T strength, bool positive);
+  // This exists soley for cereal which has friend access
```

Comment:
` which has friend access` is not meaningful anymore

Reply:
this is true!